src/ConditionalLabel.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div>
```

Comment:
Any reason why this isn't `<b-row>`, as previously? Either way, I wouldn't have though the `v-if` and `class` attributes would be better applied to this element once, than being specified twice below.

Reply:
Agreed, the row can be the parent of everything else. And should the label not have a col class as well as the dynamic form control?

Reply:
With the approach I have taken using named slots, the label or span needs to be the top level row element so that the grid structure can be properly applied. I needed to wrap them in a div because of the limitations of Vue 2.